package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@lucro/vuex",
-  "version": "0.2.5-4",
+  "version": "1.0.0",
```

Comment:
I'm confused by the diff vs what is in the PR description, is it that all the actual diff is in master already and this is just bringing the latest tag version in to match?

Reply:
This is what triggers the build/push to the npm registry